coverage_comment/settings.py

```diff
@@ -40,6 +40,7 @@ class Config:
     MINIMUM_ORANGE: float = 70.0
     MERGE_COVERAGE_FILES: bool = False
     ANNOTATE_MISSING_LINES: bool = False
+    ANNOTATION_TYPE: str = "warning"
```

Comment:
I wonder if it wouldn't make more sense to check that we're within the correct values in `clean_annotation_type` rather than later ?
(if someone installs pcca on their repo and configure it incorrectly, they'd probably prefer it to crash immediately rather than at the first uncovered line)

Reply:
Yes, you're right. That makes more sense